index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Skeleton project</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+
+  <body>
+    <header>
+      <section class="navigator1">
+        <div class="navigator">
+          <span class="mylogo">My logo</span>
+          <span><img src="images/Enabled.png" alt="" class="hamburger"></span>
+        </div>
+
+      </section>
```

Comment:
- [ ] Please section tag to structure your page to improve SEO for your website.
- [ ] The logo has a link (a tag) instead of span
- [ ] Kindly consider using the nav tag wrap hamburger menu instead of span to improve SEO.